apps/sms/lib/bridge/bridge.js

```diff
@@ -32,13 +32,27 @@ module.exports = Client;
 /**
```

Comment:
oops I forgot the activity shim again... So is it possible to replace the bridge with client in startup and replace the bridge with service/client ?